extensions/ql-vscode/package.json

```diff
@@ -354,6 +354,14 @@
         "command": "codeQL.copyVersion",
         "title": "CodeQL: Copy Version Information"
       },
+      {
+        "command": "codeQLDatabases.openConfigFile",
```

Comment:
It'd be good to keep the two database panels and their commands completely separate for now and go with `codeQLDatabasesExperimental`

Reply:
Done in https://github.com/github/vscode-codeql/pull/1719/commits/c7c0ad49cf4e665df256daf702cb6f5fa0e05894 (and https://github.com/github/vscode-codeql/pull/1719/commits/c7b217c8fa35a62a0f0bd80a06e1a002d6e42311)